x-pack/plugins/secops/public/lib/adapters/framework/testing_framework_adapter.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AppFrameworkAdapter } from '../../lib';
+
+export class AppTestingFrameworkAdapter implements AppFrameworkAdapter {
```

Comment:
I didn't see this being used anywhere or any jest like tests within it. I deleted this file from my environment and am running fine at the moment.

Reply:
Going to keep it for now I think!!!